dts/rx/renesas/rx130-common.dtsi

```diff
@@ -395,13 +395,37 @@
 			};
 		};
 
-		cmt0: timer@88004 {
-			compatible = "renesas,rx-timer-cmt";
-			reg =	<0x00088004 0x02>,
-				<0x00088000 0x02>,
-				<0x00088002 0x02>,
-				<0x00088006 0x02>;
-			status = "disabled";
+		cmt: timer@88000 {
+			compatible = "renesas,rx-timer-cmt-start-control";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			reg =	<0x00088000 0x02>;
+			reg-names = "CMSTR0";
+			status = "okay";
+
+			cmt0: timer@88002 {
+				compatible = "renesas,rx-timer-cmt";
+				#address-cells = <1>;
+				#size-cells = <1>;
+				reg =	<0x00088002 0x02>,
+					<0x00088004 0x02>,
+					<0x00088006 0x02>;
```

Comment:
remove double space, use space after tables for alignment on other lines if needed

Reply:
It's fixed